voice/bxml/callbacks/disconnect.md

```diff
@@ -1,6 +1,19 @@
 {% method %}
 ## Disconnect Event
-The Disconnect event is fired when a call ends, for any reason.
+
+The Disconnect event is fired when a call ends, for any reason. The `cause` for a disconnect event on a call can be:
```

Comment:
I think we should be clear that this list is not exhaustive and other values can appear in the future